src/stores/ui.store.ts

```diff
@@ -392,6 +392,14 @@ end tell`)
       }
       store.query = ''
     },
+    deleteTrackingProject: (id: string) => {
+      if (store.currentlyTrackedProjectId === id) {
```

Comment:
isn't this code repeated from the key handler below?

Reply:
I do not understand.
Here check if the project you are trying to delete is the same as the one you are running, first tow it and then remove it.